satpy/tests/reader_tests/test_nc_slstr.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Module for testing the satpy.readers.nc_slstr module.
+"""
+
+try:
+    import unittest.mock as mock
+except ImportError:
+    import mock
+from unittest import TestCase
+
+
+def mock_dataset():
+    """Create a mock xarray"""
+    return mock.MagicMock()
+
+
+class TestSLSTRReader(TestCase):
+
+    @mock.patch('xarray.open_dataset')
+    def test_instantiate(self, mocked_Dataset):
+
+        mocked_Dataset.return_value = mock_dataset()
+
+        from satpy.readers.nc_slstr import NCSLSTR1B, NCSLSTRGeo, NCSLSTRAngles, NCSLSTRFlag
+
+        class Dummy():
+            def __init__(self):
+                self.name = 'foo'
+
+        class Base():
```

Comment:
Can't `Dummy` be replaced with an actual DatasetID?

```python
from satpy import DatasetID
ds_id = DatasetID(name='foo')
```

And this `Base` can be replaced with a dictionary, right?

```python
filename_info = {'mission': 'S3A'}
```

Reply:
Dictionary needed a few more entries but otherwise good.